### PR TITLE
feature-NXP-19167-cmdline-pdftotext

### PR DESCRIPTION
**Pull request by mnixo**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/nuxeo/nuxeo/pull/183" target="_blank">ReviewNinja

**Comment by efge** [comment]
👍

File: nuxeo-core/nuxeo-core-convert-plugins/pom.xml [hunk 1]
```diff
@@ -33,8 +33,9 @@
       <artifactId>nuxeo-core-convert</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.nuxeo.ecm.core</groupId>
-      <artifactId>nuxeo-core-commandline-executor</artifactId>
+      <groupId>org.nuxeo.ecm.platform</groupId>
+      <artifactId>nuxeo-platform-commandline-executor</artifactId>
+      <version>8.4-SNAPSHOT</version>
```
Comment (troger) [inline nuxeo-core/nuxeo-core-convert-plugins/pom.xml]: The version should not be set here

**Comment by troger** [comment]
👍

**Pull request closed without merging**